parsers/manifest_parser.go

```diff
@@ -274,6 +274,9 @@ func (dm *YAMLParser) ComposeActions(mani *ManifestYAML, manipath string) (ar []
 			wskaction.Exec.Kind = action.Runtime
 		}
 
+                // we can specify the name of the action entry point using main
+                wskaction.Exec.Main = action.Main
```

Comment:
we should test that we can support Lambda "handlers", that is reassigning "main"

exports.main = “myAWSLambdaHandler”

which is allowed...